ssl/t1_lib.c

```diff
@@ -175,12 +179,17 @@ static const unsigned char ecformats_default[] = {
 };
 
 /* The default curves */
-static const uint16_t eccurves_default[] = {
+static const uint16_t supported_groups_default[] = {
     29,                      /* X25519 (29) */
     23,                      /* secp256r1 (23) */
     30,                      /* X448 (30) */
     25,                      /* secp521r1 (25) */
     24,                      /* secp384r1 (24) */
+    0x100,                   /* ffdhe2048 (0x100) */
+    0x101,                   /* ffdhe3072 (0x101) */
+    0x102,                   /* ffdhe4096 (0x102) */
+    0x103,                   /* ffdhe6144 (0x103) */
+    0x104,                   /* ffdhe8192 (0x104) */
```

Comment:
If we want to add this by default, I think the order is wrong, but currently don't have a good suggestion for ordering.

Reply:
Yes. I am also not having any better idea for ordering ffdhe. Requesting others to provide suggestion.

Reply:
why ffdhe2048 shouldn't be on by default? It's generally acceptable pairing with 2048 bit RSA - the dominant key size now

even if we consider it not good enough for default, shouldn't support of it depend on set security level?

Reply:
My thinking was for consistency with the other groups in the list (i.e. the lowest security bits of any other item in the list is 128). But its a fair point about RSA-2048 (which is allowed by the default security level)....so....never mind my comment.

Reply:
Added `ffdhe2048` back to this default list.